### ISSUE TITLE
[FEATURE] combine individual volumes so that 9 volumes are not created

### ISSUE THREAD

**Issue by derdek** [bunkerity/bunkerweb#163]
**What's needed and why ?**
after start bunkerized-nginx, i have 9 volumes, that was created automatic and that is not comfortable

**Implementations ideas (optional)**
I propose create one main volume, for example bunkerized-nginx-data, and put in it:
 - /acme-challenge
 - /http-confs
 - /plugins
 - /modsec-crs-confs
 - /cache
 - /server-confs
 - /pre-server-confs

**Comment by fl0ppy-d1sk** [comment]
Hello @derdek,

Correct me if I'm wrong but you would like to have the following directory on the host : 

```
/my/data/
├── acme-challenge
├── cache
├── http-confs
├── modsec-crs-confs
├── plugins
└── server-confs
...
```

And mount only one volume when running bunkerized-nginx : 
```shell
$ docker run ... -v /my/data:/volumes ... bunkerity/bunkerized-nginx
```

**Comment by derdek** [comment]
Hello, this option is suitable.
Automatically created volumes have a terrible name and are not convenient to work with.
Although it is possible to write about volumes in Readme.md that people understood whence so many volumes.

**Comment by fl0ppy-d1sk** [comment]
Hello @derdek,

Fixed in the new 1.4.0 version. Thanks for the idea.
